### DISCUSSION TITLE
How to disable autocorrection in code editor?

### DISCUSSION THREAD

**Discussion by user9931** [TriliumNext/Trilium#2931] [Q&A] [answered]
Maybe this has already been asked, and find the question...

How to disable this behaviour? Or at least configure it... Its changing !=, <=, => and maybe other. Its change only how the characters looks in editor though, and no affect on text itself

![111](https://user-images.githubusercontent.com/62539068/174541451-b2c24839-cffc-4ac1-adb5-976fc60c1248.gif)

**Comment by sigaloid** [comment, marked as answer]
These are code ligatures, I believe they're built into the font. If you change the font to a different one that doesn't have them, it shouldn't change.

**Reply by zadam** [reply]
Addition: you can configure the font in Options -> Appearance:

![image](https://user-images.githubusercontent.com/617641/174663192-367399c2-34f0-4ca1-b6ce-e76d5c42c03b.png)
